include/alpaka/workdiv/WorkDivMembers.hpp

```diff
@@ -21,6 +21,8 @@ namespace alpaka
     public:
         ALPAKA_FN_HOST_ACC WorkDivMembers() = delete;
 
+        //! Accepts different alpaka vector types and takes the last TDim number of items. TDim is explicitly specified
+        //! at the constructor call.
```

Comment:
You don't specify TDim as part of the constructor call, but as part of the type. I think this suffices:
```suggestion
        //! Accepts different alpaka vector types and takes the last TDim number of items.
```